notes/20200217_meeting.md

```diff
@@ -34,11 +34,15 @@ Other benefits of RAP include:
 ### Improved quality: 
 - no manual steps (in the part of the process selected for RAP)
 - peer review of code
-- testing (*unclear how prescreptive we want to be here*)
+- testing (*unclear how prescriptive we want to be here*)
 
 ### Increased trust:
+- code and documentation open to anyone (if possible)
+- software open to anyone?
 
 ### More efficient processes:
+- reduced time between data availability and publication
```

Comment:
Would you necessarily _need_ this to demonstrate more efficient processes, or just reduced time between data availability and publication being ready?